styles/components/alert.css

```diff
@@ -0,0 +1,32 @@
+/* css for alerts */
+.alert {
+    padding: 0.5rem 1rem;
+    margin-bottom: 1rem;
+    margin-top: 1rem;
+    border: 1px solid transparent;
+    border-radius: 0.25rem;
+}
+
+.alert-success {
+    background-color: #d4edda;
+    color: #155724;
+    border-color: #c3e6cb;
+}
+
+.alert-danger {
+    background-color: #f8d7da;
+    color: #721c24;
+    border-color: #f5c6cb;
+}
+
+.alert-warning {
+    background-color: #fff3cd;
+    color: #856404;
+    border-color: #ffeeba;
+}
+
+.alert-info {
+    background-color: #d1ecf1;
+    color: #0c5460;
+    border-color: #bee5eb;
```

Comment:
For colors, you can create variables and use them in CSS. Following this approach will help you to perform better while coding and you don't need to remember which hex color you used.
```
--background-color: #d1ecf1;
--primary-color: #0c5460;
--border-color: #bee5eb;
```

Reply:
sure thanks this helps @amarphule should i create for as many colors as i have

Reply:
yes, you can create variables as many as you needed